source/tutorials/moving-pictures.rst

```diff
@@ -436,26 +438,26 @@ We'll start by creating a feature table that contains only the gut samples. (To
      --p-where "[body-site]='gut'" \
      --o-filtered-table gut-table.qza
 
-ANCOM operates on a ``FeatureTable[Composition]`` QIIME 2 artifact, which is based on frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero. To build the composition artifact, a ``FeatureTable[Frequency]``  artifact must be provided to ``add-pseudocount`` (an imputation method), which will produce the ``FeatureTable[Composition]`` artifact.
+ANCOM-BC operates on a FeatureTable[Frequency] QIIME 2 artifact. We can run ANCOM-BC on the subject column to determine what features differ in abundance across the gut samples of the two subjects.
 
 .. command-block::
-
-   qiime composition add-pseudocount \
+   qiime composition ancombc \
      --i-table gut-table.qza \
-     --o-composition-table comp-gut-table.qza
-
-We can then run ANCOM on the ``subject`` column to determine what features differ in abundance across the gut samples of the two subjects.
-
-.. command-block::
-
-   qiime composition ancom \
-     --i-table comp-gut-table.qza \
      --m-metadata-file sample-metadata.tsv \
-     --m-metadata-column subject \
-     --o-visualization ancom-subject.qzv
+     --p-formula 'subject' \
```

Comment:
Could you address why you dont need an intercept?

Reply:
I see its addressed alittle later but it think it would be good to explain that it is alphabetical

Reply:
good call, i'll include that after the first function command block!